code/game/objects/items/devices/radio/radio.dm

```diff
@@ -385,8 +385,8 @@ GLOBAL_LIST_INIT(default_medbay_channels, list(
 	var/voicemask = 0 // the speaker is wearing a voice mask
 	var/jobname // the mob's "job"
 
-	if(jammed)
-		Gibberish_all(message_pieces, 100)
+	if(jammed && !syndiekey)
+		return //All radio is jammed, meaning everything in the jammer radius will be muted.
```

Comment:
The item needs a buff, but flat out denying all radio traffic is gonna be a no from me.